lib/lknvball_web/resolvers/event.ex

```diff
@@ -93,7 +93,16 @@ defmodule LknvballWeb.Resolvers.Event do
 
   ## CREATE EVENT MUTATION
 
-  def create_event(_, args, _) do
-    Lknvball.Events.create_event(args)
+  def create_event(_, %{input: input}, _) do
+    IO.inspect(input, label: "ARGS: ")
+    Lknvball.Events.create_event(input)
+    |> case do
+      {:ok, event} -> {:ok, %{success: true, edge: %{node: event}}} # TODO: this "edge" doesn't have a cursor.  Is it valid?
```

Comment:
a cursor will only be built when items are put onto a connection or db call, returning a single edge we dont actually care about pageInfo or shouldn't need to.

I put the edge stuff in, but didn't test it out. If its working it looks good to me

Reply:
Yeah it's working.  I'm not sure it makes sense to return an edge.  I do see why, because in theory, you can add it into the connection on the frontend.   But really, the frontend can spin up an edge if needed.  I think we should leave it as an edge for now, but I'm curious how it will work out over time.